### DISCUSSION TITLE
How do Allure reports are linked from the trend chart

### DISCUSSION THREAD

**Discussion by jvccvj** [andrcuns/allure-report-publisher#810] [General]
Hello @andrcuns! I started exploring this Allure publisher and have a doubt on how the trends chart are linked to the previous reports stored in S3.   
<img width="628" alt="Screenshot 2024-08-28 at 10 52 37" src="https://github.com/user-attachments/assets/465686c9-c363-4f7a-8c15-31c96b3e22f9"> 
`<g class="slice"><a class="edge" xlink:href="http://gitlab-qa-allure-reports.s3.amazonaws.com/e2e-package-and-test/master/1426649653/index.html"><line id="5" x1="142.89473684210526" y1="5.856000000000005" x2="142.89473684210526" y2="244" stroke="white" stroke-width="1" class="report-line"></line><rect class="report-edge" x="128.60526315789474" y="0" height="244" width="28.57894736842105" style="opacity: 0;"></rect></a></g>`
Not sure if it's the way the Allure report gets created using a convention or a post-execution manual replacement mechanism. I saw it from an example from Gitlab reports https://gitlab-qa-allure-reports.s3.amazonaws.com/e2e-package-and-test/master/index.html#
Thanks for the tool, it has been very useful so far

**Comment by jvccvj** [comment]
Nevermind, I found out that it is not directly related with this publisher duties. It is solved by the Allure report tool https://allurereport.org/docs/how-it-works-executor-file/

**Reply by andrcuns** [reply]
Well, this gem is creating the `executor.json` file to link the previous reports and create the trend graph and as long as I have been using it, the links are correct.
